.github/ISSUE_TEMPLATE/Documentation.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Documentation
+about: Report mistakes or request for documentation
+---
+[kind/Documentation]
+
+<!--
+
+Welcome! - We kindly ask that you:
+
+  1. Did you already check the documents under https://github.com/redhat-developer/odo/tree/master/docs
+  2. Use the chat and talk to us if you have a question/doubt rather than a documentat mistake or request.
+
+The chat room is at: https://chat.openshift.io/developers/channels/odo
+
+NOTE: Not filling out the issue template needs a good reason, as otherwise it
+may take a lot longer to find the problem, not to mention it can take up a lot
+more time which can otherwise be spent on development. Please also take the
+time to help us debug the issue by collecting relevant information, even if
+it doesn't seem to be relevant to you. Thanks!
+
+Thanks for understanding, and for contributing to the project!
+
+-->
+
+What did you found missing in documentation?
```

Comment:
What did you find missing in documentation ?

Reply:
Actually: What did you find missing in the documentation?